pom.xml

```diff
@@ -12,7 +12,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <dropwizard.version>0.7.1</dropwizard.version>
-        <dropwizard.cassandra.version>4.0.0</dropwizard.cassandra.version>
+        <dropwizard.cassandra.version>3.0.2</dropwizard.cassandra.version>
```

Comment:
dropwizard-cassandra 4.0.0 wasn't fully compatible with Dropwizard 0.7.1 which lead to troubles parsing the cassandra nested config in the yaml. 
Now that we've upgraded to Dropwizard 1.1.0, we can go back to using 4.0.0 (I think we're even using 4.1.0 now).